src/exodus/tx.cpp

```diff
@@ -81,6 +81,22 @@ std::string exodus::strTransactionType(uint16_t txType)
     }
 }
 
+static int EnsureEnableSigma(uint32_t propertyId)
+{
+    if (!IsPropertyIdValid(propertyId)) {
```

Comment:
This check should not be here due to the method name imply that it will check only sigma state.